opm/simulators/wells/BlackoilWellModel_impl.hpp

```diff
@@ -1991,7 +1993,9 @@ namespace Opm {
             }
             well_state.setCurrentInjectionGroupControl(group.name(), newControl);
         }
-        wellGroupHelpers::setGroupControl(group, schedule(), reportStepIdx, /*isInjector*/true, well_state, ss);
+        const auto& summaryState = ebosSimulator_.vanguard().summaryState();
+        const Phase& topPhase = group.injectionControls(summaryState).phase;
```

Comment:
I think the term "top phase" is inaccurate, is this not the "top up phase"? If so, the variable name should be `topUpPhase` here and in the function called.

Reply:
topUpPhase is on its way. See PR OPM/opm-common#1410.

Reply:
I hoped to get this merged before starting to work on the topUpPhase.

Reply:
Ok. Will you fix the typo first?